src/JSInterop/Microsoft.JSInterop/src/IJSVoidResult.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.JSInterop
```

Comment:
```suggestion
namespace Microsoft.JSInterop.Infrastructure
```

Based on the API review.